docs/onnxruntime_op.md

```diff
@@ -0,0 +1,118 @@
+# Custom operators for onnxruntime in mmcv
+
+## Introduction of onnxruntime
+
+**ONNX RUNTIME** is a cross-platform inferencing and training accelerator compatible with many popular ML/DNN frameworks. Check its [github](https://github.com/microsoft/onnxruntime) for more information.
+
+## Introduction of onnx
+
+**ONNX** stands for **Open Neural Network Exchange**, which acts as *IR(Intermediate Representation)* for ML/DNN models from many frameworks. Check its [github](https://github.com/onnx/onnx) for more information.
+
+## Why include custom operators for onnxruntime in mmcv
+
+- To support verify correctness and precision of exported onnx models in onnxruntime.
+- To support deploy onnx models with custom operators from `mmcv.ops` in onnxruntime.
```

Comment:
To ease the deployment of ONNX models with custom operators from `mmcv.ops` in ONNX runtime.